commons/pom.xml

```diff
@@ -133,7 +133,6 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <scope>test</scope>
```

Comment:
Log implementations should be `compile` only on final .jars/.wars

These are here to provide a log implementation for the tests and the scope test should not be removed